interface/skin.dmf

```diff
@@ -580,17 +580,8 @@ macro "hotkeymode"
 		name = "CTRL+E"
 		command = "quick-equip"
 	elem
-		name = "F"
```

Comment:
You still need to add back the old hotkeys to enable the cycling again.

Reply:
Huh, are the cycling hotkeys not existent further down the list? It looked like they were still there